cartridge.lua

```diff
@@ -159,12 +159,17 @@ end
 --
 -- @tparam ?number opts.bucket_count
 --  bucket count for vshard cluster. See vshard doc for more details.
+--  Could be set only **once**, before first cartridge start and coundn't be
```

Comment:
Why in the past tense? 
```suggestion
--  Can be set only **once**, before the first run of Cartridge application, and can't be
```